### PR TITLE
275817 - Use i18n for frame resource

### PR DESCRIPTION
**Pull request by MBausson**

### CODE REVIEW

File: app/views/servers/_modal_add_element_in_frame.html.erb [hunk 1]
```diff
@@ -64,7 +68,9 @@
           <div class="form-group">
             <div class="col-sm-offset-2 col-sm-10">
               <%= f.submit 'Confirmer',class: "btn btn-primary" %>
```
Comment (B-Rass) [inline app/views/servers/_modal_add_element_in_frame.html.erb:70]: Missing Translation on `Confirmer`

**Review by B-Rass — CHANGES_REQUESTED** [review]
Some miss to add

@nicolas-brousse need your opinion on one comment

File: config/locales/activerecord.fr.yml [hunk 2]
```diff
@@ -127,6 +129,11 @@ fr:
         name: Nom
         quantity: Quantité
         unit: Unité
+      frame:
+        name: Nom
+        servers: Matériels
+        room: Salle
```
Comment (nicolas-brousse) [inline config/locales/activerecord.fr.yml:135]: @MBausson still used?

**Review by nicolas-brousse — CHANGES_REQUESTED** [review]

File: config/locales/activerecord.fr.yml (hunk 2 above)
Comment (MBausson) [reply config/locales/activerecord.fr.yml:137]: Yes, in the frames index file, for the <th>

**Review by B-Rass — APPROVED** [review]

**Review by nicolas-brousse — APPROVED** [review]

File: app/models/server.rb [hunk 3]
```diff
@@ -155,7 +155,7 @@ def slug_candidates
   def numero_cannot_be_a_current_server_name
     servers = Server.friendly.where(slug: numero.to_s.downcase) - [self]
     if servers.present?
-      errors.add(:numero, "ne peut pas être identique à un nom de machine")
```
Comment (B-Rass) [inline app/models/server.rb:158]: Autant le faire maintenant nan ?

**Review by B-Rass — CHANGES_REQUESTED** [review]
1 question and good to merge @nicolas-brousse

File: app/models/server.rb (hunk 3 above)
Comment (nicolas-brousse) [reply app/models/server.rb:158]: Done

**Review by B-Rass — APPROVED** [review]

**Pull request merged**